src/components/formProducts/FormProducts.js

```diff
@@ -99,19 +97,39 @@ const FormProducts = ({ items }) => {
         </div>
         <div className="productos_view products-container">
           {itemsFiltered?.length > 0 ? (
-            itemsFiltered.map((item, index) => (
-              <ProductForList
-                key={`${index}-${item.name}`}
-                item={item}
-                token={token.current}
+            <>
+              <ProductsByEstimation
+                items={itemsFiltered}
+                start={0}
+                end={6}
+                title="Soon"
+                estimationType="soon"
               />
-            ))
+              <ProductsByEstimation
+                items={itemsFiltered}
+                start={7}
+                end={30}
+                title="Kind of Soon"
+                estimationType="kind-of-soon"
+              />
+              <ProductsByEstimation
+                items={itemsFiltered}
+                end={31}
```

Comment:
Me parece que debería ser: `start` y no así: `end`, porque es `>31 days`. Además que genera un comportamiento extraño pareciera que existiera duplicados (porque estan en mismos rango :( )

```suggestion
                start={31}
```

**Pareciera, que fueran duplicados**
![image](https://user-images.githubusercontent.com/8689897/158653253-ff37085f-e9b0-42aa-808c-9bef679ec77e.png)

**Lo esperado**

![image](https://user-images.githubusercontent.com/8689897/158653408-05d8af75-3153-4e5e-965a-1259ee4f3022.png)